lib/image.js

```diff
@@ -10,7 +11,17 @@ function createImages(files, cb) {
     // DEV: If this becomes unwieldy, load in stats only. Then, inside of exporters, stream them in individually.
     function loadImages (cb) {
       async.map(files, function loadImage (filepath, cb) {
-        getPixels(filepath, cb);
+        if (filepath.buffer) {
+          mime.lookup(filepath.buffer, function (err, info) {
+            if (err) {
+              return cb(err);
+            } else {
+              getPixels(filepath.buffer, info.mime, cb);
```

Comment:
`get-pixels` already does `mime` lookup for `fs` files. Can we submit the `mime` lookup as a PR to there instead?

https://github.com/scijs/get-pixels/blob/d975739a3ef01507a77f6c7aff37cb04f3f25cc8/node-pixels.js#L170-L175